test/trento/tags/policy_test.exs

```diff
@@ -0,0 +1,47 @@
+defmodule Trento.Tags.PolicyTest do
+  use ExUnit.Case
+
+  alias Trento.Tags.Policy
+  alias Trento.Tags.Tag
+  alias Trento.Users.User
+
+  test "should allow any action when the user has global ability" do
+    user = %User{abilities: [%{resource: "all", name: "all"}]}
+
+    Enum.each([:cluster, :host, :sap_system, :database], fn tag_resource ->
+      assert Policy.authorize(:add_tag, user, %{tag_resource: tag_resource, resource: Tag})
+    end)
+  end
+
+  test "should not allow add_tag action on the resource when the user does not have the right ability" do
+    user = %User{abilities: []}
+
+    Enum.each([:cluster, :host, :sap_system, :database], fn tag_resource ->
+      refute Policy.authorize(:add_tag, user, %{tag_resource: tag_resource, resource: Tag})
+    end)
+  end
+
+  test "should not allow delete_tag action on the resource when the user does not have the right ability" do
+    user = %User{abilities: []}
+
+    Enum.each([:cluster, :host, :sap_system, :database], fn tag_resource ->
+      refute Policy.authorize(:delete_tag, user, %{tag_resource: tag_resource, resource: Tag})
```

Comment:
This should be `remove_tag` instead of `delete_tag` and the same in the test description

Reply:
done thx, for spotting